packages/adblocker/test/serialization.test.ts

```diff
@@ -74,6 +74,26 @@ describe('Serialization', () => {
     });
   });
 
+  describe('NetworkFilter', () => {
+    it('keeps integrity with the use of 32nd bit', () => {
+      const buffer = StaticDataView.allocate(10, { enableCompression: false });
+      const mask = (1 << 31) | (1 << 30);
+      const filter = new NetworkFilter({
+        filter: undefined,
+        hostname: undefined,
+        mask,
+        domains: undefined,
+        denyallow: undefined,
+        optionValue: undefined,
+        rawLine: undefined,
+        regex: undefined,
+      });
+      filter.serialize(buffer);
+      buffer.seekZero();
+      expect(NetworkFilter.deserialize(buffer)).to.be.eql(filter);
```

Comment:
Not sure what this test does. 

Should be simply check:
```js
const filter = new NetworkFilter({
  ...
  mask: SOME_MASK
});
expect(filter.mask).to.be.eql(2147483648);
expect(filter.mask.toString(2)).to.be.eql('10000000000000000000000000000000');
```

Reply:
It keeps the integrity before/after the serialization. It's already covered by `setBit` test, I feel both are not required then.